pkg/webhook/webhook.go

```diff
@@ -256,6 +264,12 @@ func (e *Validator) handlePullRequest(ctx context.Context, pr *github.PullReques
 	sha := pr.GetPullRequest().GetHead().GetSHA()
 	installationID := pr.GetInstallation().GetID()
 
+	// Skip if the organization is not in the list of organizations to validate.
+	if e.shouldSkipOrganization(owner) {
```

Comment:
should we normalize to all lowercase in case the operator set a organization with lowercases and what we get is not, like

in the config: `myorg`
in the request `MyOrg`

Reply:
Done.